File: src/main/java/gargoyle/ct/util/ResourceException.java
package gargoyle.ct.util;

public class ResourceException extends RuntimeException {
    private static final long serialVersionUID = -695314450946813395L;

    public ResourceException(String message) {
        super(message);
    }

    public ResourceException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/main/kotlin/gargoyle/ct/util/ResourceException.kt
package gargoyle.ct.util

import java.io.Serial

class ResourceException : RuntimeException {
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        @Serial
        private val serialVersionUID = -695314450946813395L
    }
}
